aut_docs/Installation_Guide_WindowsOS.md

```diff
@@ -0,0 +1,230 @@
+## Installation Guide for Windows OS
+
+### A one-time setup:
+
+- Download [Python 3.6.5](https://www.python.org/downloads/release/python-365) (Windows x86-64 executable installer version 3.6.5) 
+  
+  **Note:** Don't forget to check **Add to path** while installing.
+
+- Check the python version 
+    ```
+       python --version
+    ```
+
+- Check if pip (The python package installer) is already installed by running
+    ```
+       pip --version
+    ```
+- If installed then upgrade it using the command
+    ```
+       python -m pip install --upgrade pip
+    ```
+  This will upgrade the pip version
+
+- Install virtual environment
+    ```
+       pip install virtualenv
+    ```
+- Download [Git](https://git-scm.com/download/win) 
+  
+  **Note:** While installing select the option **Use Git and optional Unix Tools from the Windows Command Prompt** for using Git from the command prompt
+ 
+- Clone the VMS project
+    ```
+       git clone https://github.com/systers/vms.git
```

Comment:
Update clone link to` git clone https://github.com/anitab-org/vms`

Reply:
ok I will update it but I guess using the above also clones it correctly?